Source/SwiftLintBuiltInRules/Rules/Style/IndentationStyleRule.swift

```diff
@@ -0,0 +1,385 @@
+import Foundation
+import SourceKittenFramework
+
+struct IndentationStyleRule: ConfigurationProviderRule, OptInRule {
+    // MARK: - Subtypes
+    private enum Indentation: Equatable {
+        case tabs
+        case spaces
+    }
+
+    // MARK: - Properties
+    var configuration = IndentationStyleConfiguration()
+
+    static let description = RuleDescription(
+        identifier: "indentation_style",
+        name: "Indentation Style",
+        description: """
+            Indent code using either tabs or spaces, not a chaotic mix. Can be configured to per file or project wide.
```

Comment:
I had to laugh about the "chaotic mix". Yet, I think it's a bit too offensive for a documentation. Better leave it out. 😉 

```suggestion
            Code should be indented using either tabs or spaces. Can be configured to per file or project wide.
```

Reply:
> I had to smile when I read the "chaotic mix"

Mission accomplished :) I can now close this PR!